src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Host, h, Prop, Watch, State } from "@stencil/core";
+
+@Component({
+  tag: "radio-group",
+  styleUrl: "radio-group.css",
+  shadow: true,
+})
+export class RadioGroup {
+  @Prop({ reflect: true }) list: Array<string>;
+  @State() selectedIndex: number;
+  @State() selectedItem: string;
+
+  handleChange = (e: any) => {
+    this.selectedIndex = parseInt(e.target.value);
+    this.selectedItem = e.target.name;
+  };
+
+  render() {
+    return (
+      <Host>
+        <slot>
```

Comment:
Can you explain why you added a default slot tag here? Have you tried slotting elements inside of this component?

Reply:
We don't need the slot element here.